### PR TITLE
Attempt frontend deploy

### PR DESCRIPTION
**Pull request by cloudyyoung**

### CODE REVIEW

**Review by tim-macphail — APPROVED** [review]
Not sure how I can test the yaml files but it looks good to me👍

File: client/src/pages/Landing.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Link } from "react-router-dom";
+
+export default function About() {
```
Comment (jaza-k) [inline client/src/pages/Landing.tsx:3]: ```export default function Landing()``` instead of ```About()```?

File: client/src/pages/Loading.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { Link } from "react-router-dom";
+
+export default function About() {
```
Comment (jaza-k) [inline client/src/pages/Loading.tsx:3]: Change ```About()``` to ```Loading()```

File: client/src/pages/Review.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { Link } from "react-router-dom";
+
+export default function Home() {
```
Comment (jaza-k) [inline client/src/pages/Review.tsx:3]: Rename function component

File: client/tsconfig.json [hunk 4]
```diff
@@ -1,11 +1,7 @@
 {
   "compilerOptions": {
     "target": "es5",
-    "lib": [
```
Comment (jaza-k) [inline client/tsconfig.json:4]: I normally leave it on multiple lines for config files, better readability but just a thought!

**Review by jaza-k — COMMENTED** [review]
Minor changes

File: client/tsconfig.json (hunk 4 above)
Comment (cloudyyoung) [reply client/tsconfig.json:4]: This is probably done by the linter

File: client/src/pages/Landing.tsx (hunk 1 above)
Comment (cloudyyoung) [reply client/src/pages/Landing.tsx:3]: These would be fixed as soon as someone starts on the page

**Comment by cloudyyoung** [comment]
Hey @jaza-k I'm trying to merge something to production branch so that GitHub Action would be triggered to perform deploy frontend, as we want to know if the workflow is working and we could fix it if not. If you could quickly approve this pr and I'll merge it, that would be appreciated!

**Review by jaza-k — APPROVED** [review]
LGTM

**Pull request merged**